jujugui/static/gui/src/app/components/profile/bundle-list/bundle-list.js

```diff
@@ -200,6 +216,7 @@ class ProfileBundleList extends React.Component {
                     {version}
                   </div>
                 </div>)} />),
+          extraData: bundle.name,
```

Comment:
what about 'sortData'?

Reply:
This is the key the table finds the extra sort info from.